acceptance/acceptance_test.go

```diff
@@ -1561,18 +1590,29 @@ include = [ "*.jar", "media/mountain.jpg", "media/person.png" ]
 				var buildRunImage func(string, string, string)
 
 				it.Before(func() {
+					pack.JustRunSuccessfully("trust-builder", builderName)
```

Comment:
I'm guessing this was needed because we don't support untrusted builder workflows on Windows yet (though I'm confused how it relates to rebase, could you explain?). 

I'm a bit concerned about this setup because it could potentially pollute other tests (it modifies that pack config file). Is there any way to pass `--trust-builder` instead?

Reply:
I understand the confusion. Trusting the builder is needed for the reason you mentioned, but we added this line because `when("rebase")` is a sibling to the `when("build")` where this normally happens (under `when("default builder is set")`, so we duplicated it here. I don't feel strongly one way or the other regarding `--trust-builder` instead, though the pollution argument would be a problem already (due to the `when("build")`.

Reply:
Oh, ha! I guess I'm wondering, if you remove the setup, do the rebase tests still pass? I would assume so, but maybe I'm wrong...

Reply:
@natalieparellano If this were removed, the tests would pass in the Linux container case and fail in the Windows container case

Reply:
Ohhh... I think I get it now. I was struggling to see how "trusted builder" has anything to do with rebase - but it looks like the test setup has a `pack build` that would fail if the builder is not trusted. Maybe it's worth adding a comment, so that others don't also get confused. Or you could use `--trust-builder` just where it's needed.